src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -617,7 +632,8 @@ def fuzz(self,
     with open(self.fuzzer.logfile) as logfile:
       symbolized_output = logfile.read()
 
-    self._pull_new_corpus_from_target_to_host(corpus_directories)
+    self._pull_new_corpus_from_target_to_host(corpus_directories,
```

Comment:
Can we move this to before line 631? Then we can get rid of the crash artifact copying logic in `process_logs_and_crash` which is otherwise duplicated.